AWSCoreTests/AWSTestUtility.m

```diff
@@ -65,6 +65,21 @@ + (void)initialize {
     [super initialize];
 }
 
++ (void) setupSTSBasedSessionCredentialsProvider {
+    NSString *filePath = [[NSBundle bundleForClass:[self class]] pathForResource:@"credentials"
```

Comment:
Doesn't have to be in this PR, but let's change the name of the file that gets written from `credentials.json` to `testconfiguration.json`, to be more accurate. It has the added benefit of aligning with Android.

Reply:
noted

Reply:
noted